### DISCUSSION TITLE
Is the alert_notification email template generated?

### DISCUSSION THREAD

**Discussion by rosalinddeibert** [grafana/grafana#68583] [General]
The README in `public/emails/` says "html files in this folder are generated from templates and build system in repo_root/emails".

There are templates in `emails/templates/` that can be used to generate new html files, and which correspond to all of the email templates in `public/emails`, except for alert_notification.

Does anybody know if the alert_notification template in `public/emails` is being generated from somewhere? And if not, is there a reason it should not be generated from an mjml template in `emails/` along with the rest of the email templates?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
